solr/contrib/scripting/src/test/org/apache/solr/scripting/xslt/XSLTUpdateRequestHandlerTest.java

```diff
@@ -39,11 +38,17 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
-public class XsltUpdateRequestHandlerTest extends SolrTestCaseJ4 {
-  
+/**
+ * <p>
+ * This tests the XSLTUpdateRequestHandler ability to work with XSLT stylesheet and xml content.
+ * </p>
+*/
+public class XSLTUpdateRequestHandlerTest extends SolrTestCaseJ4 {
```

Comment:
BTW, I the most common naming convention is for acronyms to use camel case instead of all-caps.  This has been a long trend over many years.  I know why you changed it here -- to match the class it's testing, and so it's a net improvement over what existed.  But arguably it's the class it's testing that should be renamed; not this.

Reply:
I am down for making this change...  XSLT -> Xslt for the files touched in this PR, and then maybe a seperate JIRA for XML --> Xml  in the other classes?